### DISCUSSION TITLE
Removing prohibited props from a building

### DISCUSSION THREAD

**Discussion by NI0FREE** [wiremod/advdupe2#419] [Q&A]
I'm playing on Garry's mod server and I have buildings saved via advanceddupe2.
Recently, the server has added some props to the blacklist and now, when installing old buildings, advdupe cannot build them, that is, when the queue reaches the forbidden props, it simply stops building.
How can I remove forbidden props or just skip their construction? The construction file itself looks strange, I can't figure out how to decode it and delete the forbidden propes. Please tell me what can be done with this.

**Comment by thegrb93** [comment]
If you have the latest version of advdupe2, it should continue despite the blacklisted props.

**Reply by NI0FREE** [reply]
Thanks for the answer, however, as stated earlier, the server stops construction for some reason. Perhaps this is due to internal plugins or an old version of advdupe2. And that's why I can't find a solution to this problem, it's not possible to edit the file, maybe there are other ways to fix the file?

**Reply by NI0FREE** [reply]
I add: I went into single mode and deleted the conflicting props. As a result, when opening the file of the building on the server, the duplicator breaks down and opens the file indefinitely, other files also do not open after trying to open a new file

**Reply by thegrb93** [reply]
It sounds like the server has an old version
